released/packages/coq-fiat-crypto/coq-fiat-crypto.0.0.15/opam

```diff
@@ -20,7 +20,7 @@ depends: [
   "ocamlfind" {build}
   "coq" {>= "8.15~"}
   "coq-coqprime" {>= "1.2.0"}
-  "coq-rewriter" {= "0.0.6"}
+  "coq-rewriter" {>= "0.0.6" & <= "0.0.7"}
```

Comment:
@MSoegtropIMC This sort of bounds relaxation on platform packages is okay, right?